storage/errors/error.go

```diff
@@ -0,0 +1,14 @@
+package errors
+
+import "errors"
+
+var (
+	// NotInitialized indicates storage instance was not correctly initialized and contains empty required values.
```

Comment:
suggestion: A convention we use in flow-go, which is used elsewhere is to prefix static errors with `Err`, e.g. `ErrNotInitialized`. this makes it easy to differentiate static errors from error types.